Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -364,6 +355,9 @@ - (UIBarButtonItem*)shareToolbarItem {
 - (void)viewDidLoad {
     [super viewDidLoad];
 
+    [self setupToolbar];
```

Comment:
@coreyfloyd the `WMFAppViewController` [checks `toolbarItems` and hides/unhides it manually](https://github.com/wikimedia/wikipedia-ios/blob/master/Wikipedia/UI-V5/WMFAppViewController.m#L444).  if you want to get into refactoring that, it's fine, but I don't think it's necessary for this ticket if you keep initially toolbar setup in `init`, which is unlikely to have a negative impact IMO.